### PR TITLE
Need more AIs

### PR DESCRIPTION
**Pull request by kariato**
Please review this, @casey-collab

### CODE REVIEW

File: english/turing-test/turing-test.md [hunk 1]
```diff
@@ -24,4 +24,8 @@ Do you:
 
 [Forget ELIZA, ask Siri](siri/ask_siri.md)
 
-[Wear the one ring and turn invisible](ring/one_ring.md)
\ No newline at end of file
+[Wear the one ring and turn invisible](ring/one_ring.md)
+
+[Forget Siri, asks Cortana](Cortana/ask_Cortana.md)
```
Comment (casey-collab) [inline english/turing-test/turing-test.md]: It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.

File: english/turing-test/Cortarna/ask_Cortana.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
+Repeat You launch Cortana
```
Comment (casey-collab) [inline english/turing-test/Cortarna/ask_Cortana.md]: It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by kariato** [comment]
Please review this, @casey-collab

File: english/turing-test/Cortarna/ask_Cortana.md (hunk 2 above)
Comment (casey-collab) [inline english/turing-test/Cortarna/ask_Cortana.md]: It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Pull request closed without merging**